### PR TITLE
Grouped MEI's with insertions in splitvariants.py

### PR DESCRIPTION
**Pull request by kirtanav98**
This address issue [649](https://github.com/broadinstitute/gatk-sv/issues/649). svtk vcf2bed uses the ALT field to produce the svtype column in the output BED file. This means that the svtype column includes BND alt alleles and values like INS:ME for MEIs. However, the [current](https://github.com/broadinstitute/gatk-sv/blob/0cad1662db0522dee17ae6d874a51cab6b739cae/src/sv-pipeline/04_variant_resolution/scripts/split_variants.py#L20) and [previous](https://github.com/broadinstitute/gatk-sv/blob/v0.28.4-beta/wdl/TasksGenotypeBatch.wdl#L44) SplitVariants tasks in GenotypeBatch matched exactly on the string "INS" when creating insertion-specific BED files, so the MEIs were grouped with BCAs instead. Here the MEI's are grouped together with the insertions when creating the insertion-specific BED files instead of the BCA's. This can allow for further evaluation the impact of this on genotyping. This has been successfully been validated with womtool and cromshell using the 1kgp reference panel inputs. The results of the previous script and docker and the results of the updated script and docker can be found [here](https://console.cloud.google.com/storage/browser/broad-dsde-methods-kv/splitvariants_ins?authuser=0&project=broad-dsde-methods&pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22)))

### CODE REVIEW

File: inputs/values/dockers.json
Comment (epiercehoffman) [inline inputs/values/dockers.json:1]: Please revert these changes - the dockers will get built and updated automatically when this is merged!

File: inputs/values/dockers.json [hunk 1]
```diff
@@ -29,4 +29,4 @@
   "gq_recalibrator_docker": "us.gcr.io/broad-dsde-methods/markw/gatk:mw-tb-form-sv-filter-training-data-899360a",
   "str": "us.gcr.io/broad-dsde-methods/gatk-sv/str:2023-05-23-v0.27.3-beta-e537bdd6",
   "denovo": "us.gcr.io/broad-dsde-methods/gatk-sv/denovo:2024-06-04-v0.28.5-beta-a8dfecba"
-}
\ No newline at end of file
+}
```
Comment (mwalker174) [inline inputs/values/dockers.json:32]: You should revert this trivial change as well. You can be sure by using `git checkout main inputs/values/dockers.json`

**Review by mwalker174 — APPROVED** [review]
Thank you for catching this bug @kirtanav98. The genotyping from your tests looks good. Here is a summary of fraction of variants in HWE before -> after the changes by ALT allele:

- `<INS>`: 0.50 -> 0.50
- `<INS:ME:ALU>`: 0.52 -> 0.71
- `<INS:ME:SVA>`: 0.56 -> 0.76
- `<INS:ME:LINE1>`: 0.59 -> 0.75

Just one small request.

File: inputs/values/dockers.json
Comment (kirtanav98) [reply inputs/values/dockers.json:1]: these were reverted

File: inputs/values/dockers.json (hunk 1 above)
Comment (kirtanav98) [reply inputs/values/dockers.json:32]: This should be fixed. Thank you!
